acme_tiny.py

```diff
@@ -112,8 +112,7 @@ def _poll_until_not(url, pending_statuses, err_msg):
     log.info("{0} Account ID: {1}".format("Registered!" if code == 201 else "Already registered!", acct_headers['Location']))
     if contact is not None:
         account, _, _ = _send_signed_request(acct_headers['Location'], {"contact": contact}, "Error updating contact details")
-        log.info("Updated contact details:\n{0}".format("\n".join(account['contact'])))
-
+        log.info("Updated contact details:\n{0}".format("\n".join(contact)))
```

Comment:
Why not use
```suggestion
        log.info("Updated contact details:\n{0}".format("\n".join(account.get('contact') or [])))
```
?

That way the user at least can see that Let's Encrypt didn't store the contact info.

Reply:
I'm not that comfortable with python (at least anymore) and this was just a quick and dirty fix. So yeah 😅

Reply:
Can we get this approved and merged?

Reply:
I will review it soon (have been on vacation)